src/target/stream.c

```diff
@@ -111,17 +163,44 @@ new_stream_target( FILE *stream ) {
 int
 sendto_stream_target( struct stream_target *target,
                       const char *msg,
-                      size_t msg_length ) {
+                      size_t msg_length,
+                      const struct stumpless_entry *entry ) {
   size_t fwrite_result;
+  enum stumpless_severity severity = stumpless_get_entry_severity(entry);
+  const char *sev_code = target->escape_codes[severity];
+  unsigned short sev_code_len = strlen(sev_code);
+  const char *reset_code = "\33[0m";
+  unsigned short reset_code_len = strlen(sev_code);
```

Comment:
should the argument to `strlen` be `reset_code` here instead?

Reply:
Yes. Done.